index.bs

```diff
@@ -1080,10 +1080,16 @@ Reads back the {{MLTensor/[[data]]}} of an {{MLTensor}} from the {{MLContext}}.{
     1. If |tensor|.{{MLTensor/[[isDestroyed]]}} is true, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
     1. If |tensor|.{{MLTensor/[[descriptor]]}}.{{MLTensorDescriptor/readable}} is false, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
     1. Let |promise| be [=a new promise=].
+    1. [=set/Append=] |promise| to |tensor|.{{MLTensor/[[pendingPromises]]}}.
     1. Enqueue the following steps to |tensor|.{{MLGraph/[[context]]}}.{{MLContext/[[timeline]]}}:
         1. Let |bytes| be a [=/byte sequence=] containing a copy of |tensor|.{{MLTensor/[[data]]}}.
-        1. If that fails, then [=queue an ML task=] with |global| to [=reject=] |promise| with an "{{UnknownError}}" {{DOMException}}, and abort these steps.
-        1. Otherwise, [=queue an ML task=] with |global| to [=ArrayBuffer/create=] an {{ArrayBuffer}} |result| given |bytes| and |realm| and then [=resolve=] |promise| with |result|.
+        1. If that fails, then [=queue an ML task=] with |global| and the following steps:
+            1. [=set/Remove=] |promise| from |tensor|.{{MLTensor/[[pendingPromises]]}}.
+            1. [=Reject=] |promise| with an "{{UnknownError}}" {{DOMException}}.
+        1. Otherwise, [=queue an ML task=] with |global| and the following steps:
+            1. [=set/Remove=] |promise| from |tensor|.{{MLTensor/[[pendingPromises]]}}.
+            1. Let |buffer| be the result of [=ArrayBuffer/creating=] an {{ArrayBuffer}}  from |bytes| in |realm|.
```

Comment:
total nitpick: remove the extra space before "from"

... and bonus points if you add a lint rule that would catch it. 😉

Reply:
Removed double-space. I'll explore a lint rule in a follow-up.. :P